feat: 32 nutzerdetails editieren

lisa-q opened the pull request:
- Edit member names
- Implemented scrolling for Sidebar
- Improved boxes with member names so that buttons and text fields do not overlap
- Added pop up to inform user that when editing the member name it is not possible to leave the text field empty

Commit:
Added user name editing functionality

AlexInABox commented:
Am besten mergen wir #47 noch vor dieser PR hier! Dann würde nämlich das Feature hier auch als Mobile App Version veröffentlicht werden ^^

Commit:
New changes:
- Edit member names
- Implemented scrolling for Sidebar
- Improved boxes with member names so that buttons and text fields do not overlap
- Added pop up to inform user that when editing the member name it is not possible to leave the text field empty

AlexInABox committed:
Merge branch 'dev' into 32-nutzerdetails-editieren

AlexInABox committed:
version bump
